modules/validations/oneway_onelane.js

```diff
@@ -0,0 +1,71 @@
+import { actionChangeTags } from '../actions/change_tags';
+import { t } from '../core/localizer';
+import { validationIssue, validationIssueFix } from '../core/validation';
+import { utilDisplayLabel } from '../util';
+
+
+export function validationOneLaneWithNoOneway() {
+  const type = 'oneway_onelane';
+
+  /** @param {string} onewayValue  @param {osmEntity} entity */
+  function createSuggestion(onewayValue, entity) {
+    return new validationIssueFix({
+      icon: 'iD-icon-data',
+      title: t.append(`issues.fix.tag_as_oneway_${onewayValue}.title`),
+      onClick: (context) => {
+        const newTags = { ...entity.tags, oneway: onewayValue };
+        context.perform(
+          actionChangeTags(entity.id, newTags), t(`issues.fix.tag_as_oneway_${onewayValue}.annotation`)
+        );
+      }
+    });
+  }
+
+  /** @param {osmEntity} entity */
+  function makeOneLaneIssue(entity) {
+    return new validationIssue({
+      type,
+      subtype: type,
+      severity: 'warning',
+      message: (context) => t.append(
+        'issues.oneway_onelane.message',
+        { feature: utilDisplayLabel(entity, context.graph()) }
+      ),
+      reference: showReference,
+      entityIds: [entity.id],
+      hash: type,
+      dynamicFixes: () => [
+        // suggest adding `oneway` tag (automatic)
+        ...['yes', 'alternating'].map(onewayValue => createSuggestion(onewayValue, entity)),
+
+        // or changing the `lanes` tag (manual)
```

Comment:
Should we add a suggestion of adding `lane_markings=no`, for unstriped two-way roads?

Reply:
sounds good, I've made this change